flutter-studio/src/io/flutter/module/FlutterModuleImporter.java

```diff
@@ -169,7 +169,6 @@ private static void showHowToEditDialog() {
                          "edit the files as described in the <a href=\"" +
                          EDIT_INSTR_LINK +
                          "\">add2app documentation.</a></body></html>";
-      Messages.installHyperlinkSupport(myTextPane);
```

Comment:
We don't need this anymore?

Reply:
Correct, I forgot to remove this line after copying it further up the method.